typings/resin-device-config.d.ts

```diff
@@ -23,16 +23,13 @@ declare module 'resin-device-config' {
 			subscribe_key?: string;
 			publish_key?: string;
 		};
-		mixpanel: {
-			token?: string;
-		};
 		apiKey?: string;
 		version?: string;
 	}
 
 	export interface GenerateParams {
 		appUpdatePollInterval?: number;
-		network?: Array<>;
+		network?: any[];
```

Comment:
@dfunckt I need your help with the build error we get here.

According to the balenaOS docks, the network field in config.json is an object
https://github.com/balena-os/meta-balena#network

How come we have an array in the type definition here? Also, in device-config implementation, it's filled with `req.param`. What is this request parameter expected value?

Reply:
If you rebase this I think I already fixed it